clients/client-python/gravitino/api/credential/adls_token_credential.py

```diff
@@ -0,0 +1,90 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from abc import ABC
+from typing import Dict
+
+from gravitino.api.credential.credential import Credential
+from gravitino.utils.precondition import Precondition
+
+
+class ADLSTokenCredential(Credential, ABC):
+    """Represents ADLS token credential."""
+
+    ADLS_SAS_TOKEN_CREDENTIAL_TYPE: str = "adls-sas-token"
+    ADLS_DOMAIN: str = "dfs.core.windows.net"
+    _GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME: str = "azure-storage-account-name"
+    _GRAVITINO_ADLS_SAS_TOKEN: str = "adls-sas-token"
+
+    def __init__(self, credential_info: Dict[str, str], expire_time_in_ms: int):
+        self._account_name = credential_info[self._GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME]
```

Comment:
Please always be cautious when calling `data[field]` because we never know if the `field` is a member of `data`. A better practice would be `data.get(field, None)`. The latter never fails with an exception.

Reply:
yes, data.get(field) seems enough, it will return None if the field not exists

Reply:
Thanks